script/update-git-lfs.js

```diff
@@ -123,11 +130,12 @@ async function run() {
   for (const file of files) {
     const re = new RegExp(`([0-9a-z]{64})\\s*${file}`)
     const match = re.exec(fileContents)
+    const platform = getPlatform(file)
     if (match == null) {
       console.log(`🔴 Could not find entry for file ${platform}`)
```

Comment:
`tsc` also found this bug because `platform` wasn't actually defined when I tried to use it here.

Lifting it up from the `else` path here resolves this problem.